sechub-wrapper-xray/src/main/java/com/mercedesbenz/sechub/wrapper/xray/util/InputStreamSaver.java

```diff
@@ -2,12 +2,12 @@
 
 import java.io.*;
 
+import com.mercedesbenz.sechub.wrapper.xray.XrayWrapperException;
 import com.mercedesbenz.sechub.wrapper.xray.cli.XrayWrapperExitCode;
-import com.mercedesbenz.sechub.wrapper.xray.cli.XrayWrapperRuntimeException;
 
 public class InputStreamSaver {
 
-    public static void saveInputStreamToZipFile(String filename, InputStream inputStream) throws XrayWrapperRuntimeException {
+    public static void saveInputStreamToZipFile(String filename, InputStream inputStream) throws XrayWrapperException {
```

Comment:
Why are all these methods static?
Why not just use an instance (maybe shared/default one).

For testing purposes static methods are most time problematic.